pkg/flag/scan_flags.go

```diff
@@ -69,36 +69,39 @@ func (f *ScanFlagGroup) Flags() []*Flag {
 	return []*Flag{f.SkipDirs, f.SkipFiles, f.OfflineScan, f.SecurityChecks}
 }
 
-func (f *ScanFlagGroup) ToOptions(args []string) ScanOptions {
+func (f *ScanFlagGroup) ToOptions(args []string) (ScanOptions, error) {
 	var target string
 	if len(args) == 1 {
 		target = args[0]
 	}
+	securityChecks, err := parseSecurityCheck(getStringSlice(f.SecurityChecks))
+	if err != nil {
+		return ScanOptions{}, xerrors.Errorf("unable to parse security checks: %w", err)
+	}
 
 	return ScanOptions{
 		Target:         target,
 		SkipDirs:       getStringSlice(f.SkipDirs),
 		SkipFiles:      getStringSlice(f.SkipFiles),
 		OfflineScan:    getBool(f.OfflineScan),
-		SecurityChecks: parseSecurityCheck(getStringSlice(f.SecurityChecks)),
-	}
+		SecurityChecks: securityChecks,
+	}, nil
 }
 
-func parseSecurityCheck(securityCheck []string) []string {
+func parseSecurityCheck(securityCheck []string) ([]string, error) {
 	switch {
 	case len(securityCheck) == 0: // no checks
-		return nil
+		return nil, xerrors.New("no security checks")
```

Comment:
`security-checks` can be empty when generating SBOM

Reply:
I didn't think about it. Thanks. Fixed.